pallets/elections/src/impls.rs

```diff
@@ -226,13 +225,34 @@ where
         )
     }
 
+    pub fn ban_expired(start: EraIndex, period: EraIndex, active_era: EraIndex) -> bool {
+        start + period < active_era
+    }
+
     fn if_era_starts_do<F: Fn()>(era: EraIndex, start_index: SessionIndex, on_era_start: F) {
         if let Some(era_start_index) = T::EraInfoProvider::era_start_session_index(era) {
             if era_start_index == start_index {
                 on_era_start()
             }
         }
     }
+    fn unban_on_new_era_start(session: SessionIndex) {
```

Comment:
can we call it `erase_expired_bans_on_new_era_start`?

Reply:
changed.